cmd/consensus/main.go

```diff
@@ -69,6 +69,7 @@ func main() {
 		blockRateDelay                         time.Duration
 		requireOneApproval                     bool
 		chunkAlpha                             uint
+		requiredChunkApprovals                 uint
```

Comment:
These two flags are redundant. I would suggest keeping `requiredChunkApprovals` and replace any usage of `requireOneApproval` with the evaluation of `requiredChunkApprovals > 0`

Reply:
yeah, makes sense, thanks